### PR TITLE
integrate yelp api; do housekeeping

### PR DESCRIPTION
**Pull request by kkoeckritz**
There may be some minor conflicts since I've done some housekeeping. My routeing etc will have to be merged into Nguyen's files later

### CODE REVIEW

**Comment by mfamis** [comment]
> Can I just ignore package-lock.json? We'll have to do a fresh `npm i` after merging anyway.

@kkoeckritz You can probably add it to the .gitignore (and remove it). The package.json should be sufficient.

**Comment by kkoeckritz** [comment]
Axed package-lock and merged api routes.

**Comment by kkoeckritz** [comment]
> 
> 
> > Can I just ignore package-lock.json? We'll have to do a fresh `npm i` after merging anyway.
> 
> @kkoeckritz You can probably add it to the .gitignore (and remove it). The package.json should be sufficient.

Done.

File: routes/api.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 // Dependencies
```
Comment (mfamis) [inline routes/api.js]: My preference would be to have the yelp endpoints in their own module (e.g. routes/yelp.js) and have use import them in server how we do api and html (require("./routes/yelp)(app)). I'll let you determine if that's how we want to organize the code.

**Pull request merged**